core/session.ts

```diff
@@ -66,7 +66,37 @@ function mapColumnDescriptionToProto(
   );
 }
 
-export type Resolvable = string | { schema: string; name: string };
+export type FullyQualifiedName = { schema: string; name: string };
+export type Resolvable = string | FullyQualifiedName;
+
+export function isResolvable(res: any) {
+  return typeof res === "string" || (!!res.schema && !!res.name);
+}
+
+export function resolvable2string(res: Resolvable) {
+  return typeof res === "string" ? res : `${res.schema}.${res.name}`;
+}
+
+export function resolvable2FQame(res: Resolvable) {
```

Comment:
doesn't appear to be used anywhere, remove?

Reply:
Done